music_assistant/providers/audiobookshelf/abs_schema.py

```diff
@@ -314,3 +320,117 @@ class ABSLibrariesItemsResponse(BaseModel):
     """
 
     results: list[ABSLibraryItem]
+
+
+# Schema to enable sessions:
+@dataclass
+class ABSDeviceInfo(BaseModel):
+    """ABSDeviceInfo.
+
+    https://api.audiobookshelf.org/#device-info-parameters
+    https://api.audiobookshelf.org/#device-info
+    https://github.com/advplyr/audiobookshelf/blob/master/server/objects/DeviceInfo.js#L3
+    """
+
+    device_id: Annotated[str, Alias("deviceId")] = ""
```

Comment:
ha cool, you are using these new helpers from mushumaro to translate from/to json.
Nice!